jinahub/encoders/image/ImagePaddlehubEncoder/Dockerfile.gpu

```diff
@@ -0,0 +1,20 @@
+FROM nvidia/cuda:10.1-cudnn7-runtime
+
+RUN apt update && DEBIAN_FRONTEND=noninteractive apt install -y python3.8 python3.8-distutils git libgomp1 libglib2.0-0 libsm6 libxext6 libfontconfig1 libxrender1 wget
+RUN wget https://bootstrap.pypa.io/get-pip.py
+RUN python3.8 get-pip.py
```

Comment:
Why not just add `python3-pip` ?

Reply:
This did not work in the container. It always uninstalled python3.8 and setup 3.6 instead. It has somethind to do with the cuda container.... This way it works